src/graphql/node.rs

```diff
@@ -461,23 +462,23 @@ impl NodeStatus {
     async fn id(&self) -> ID {
         ID(self.id.to_string())
     }
-
+    /// The RAM size in bytes within the range representable by a `u64`.
     async fn total_memory(&self) -> Option<StringNumber<u64>> {
         self.total_memory.map(StringNumber)
     }
-
+    /// The amount of used RAM in bytes within the range representable by a `u64`.
     async fn used_memory(&self) -> Option<StringNumber<u64>> {
         self.used_memory.map(StringNumber)
     }
-
+    /// The total disk space in bytes within the range representable by a `u64`.
     async fn total_disk_space(&self) -> Option<StringNumber<u64>> {
         self.total_disk_space.map(StringNumber)
     }
-
+    /// The total disk space in bytes that is currently used within the range representable by a `u64`.
     async fn used_disk_space(&self) -> Option<StringNumber<u64>> {
         self.used_disk_space.map(StringNumber)
     }
-
+    /// The ping value for a specific node within the range representable by a `i64`.
```

Comment:
You can find out useful information what the ping value exactly means in the code such as the following in the changelog:

```
 - `AgentManager::ping` method to measure the latency between the agent manager
  and a host.
```
Another one is written in the `ping` method as follows:
```
    /// Sends a ping message to the given host and waits for a response. Returns
    /// the round-trip time in microseconds.
    async fn ping(&self, _hostname: &str) -> Result<i64, anyhow::Error>;
```
From these, I suggest like:
```
/// The round-trip time in microseconds to a host, within the range representable by an `i64`.
```

Reply:
I am obviously not Sehkone, but if I may chip in, it seems fine to me to make change only for "ping". My understanding of the reasons behind Kone's request is to:
- indicate the unit as microsecond
- be kind to explain ping as round-trip time, because ping might be a bit unfriendly term for client side engineers

Documentation comments on other fields seem to be meeting these criteria already, so I think just modifying the documentation comment for "ping" is enough.